lab27/src/Vimeo.js

```diff
@@ -0,0 +1,15 @@
+import React, { useEffect } from 'react';
+import Player from '@vimeo/player';
+
+const Vimeo = ({ id = 19231868, width = 640 }) => {
+  let player = null;
+  useEffect(() => {
```

Comment:
What happens when a new id is passed as prop?

Reply:
i didn't make the video change, i know... wanted to exercise other things. once i have this component
using the props and / or useEffect to change the video is not difficult